slack_connector.py

```diff
@@ -649,6 +651,66 @@ def _create_channel(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS, SLACK_SUCCESSFULLY_CHANNEL_CREATED)
 
+    def _get_channel(self, param):
+        action_result = self.add_action_result(phantom.ActionResult(dict(param)))
+        searchbyname = False
+        searchbyid = False
+
+        channel_types = []
+        if param.get("private_channels"):
+            channel_types.append("private_channel")
+        if param.get("public_channels"):
+            channel_types.append("public_channel")
+        if param.get("name"):
+            endpoint = "conversations.list"
+            searchbyname = True
+        if param.get("id"):
+            endpoint = "conversations.info"
+            searchbyid = True
+
+        if (searchbyname == False) and (searchbyid == False):
+            return action_result.set_status(phantom.APP_ERROR)
```

Comment:
We can give the user more info here

```suggestion
            return action_result.set_status(phantom.APP_ERROR, "Must search by one of IDs or names")
```